fastcrypto-tbls/benches/dkg.rs

```diff
@@ -67,7 +67,7 @@ mod dkg_benches {
                     |b| b.iter(|| d0.create_message(&mut thread_rng())),
                 );
 
-                let message = d0.create_message(&mut thread_rng());
+                let message = d0.create_message(&mut thread_rng()).unwrap();
```

Comment:
Add a note why this `unwrap` is justified (or use `except`). I think this only happens if the weight is zero.

Same for line 88 and in nidkg.rs.